app/src/main/res/layout/fragment_beneficiary_application.xml

```diff
@@ -78,8 +78,9 @@
                     android:gravity="start"
                     android:hint="@string/transfer_limit"
                     android:imeOptions="actionNext"
-                    android:inputType="numberDecimal"
+                    android:inputType="number"
```

Comment:
I have made changes here, as decimal values causes crash, because https://github.com/openMF/mifos-mobile/blob/development/app/src/main/java/org/mifos/mobilebanking/ui/fragments/BeneficiaryApplicationFragment.java#L247.

Or should I add Double in the BeneficiaryPayload for `transfer Limit`.

Reply:
Yes, you should add the double in BeneficiaryPayload payload and making it integer is not valid.